libs/model/src/contest/Contests.projection.ts

```diff
@@ -284,7 +284,7 @@ export function Contests(): Projection<typeof inputs> {
 
       // This happens for each recurring contest _after_ the initial contest
       ContestStarted: async ({ payload }) => {
-        const contest_id = payload.contest_id || 0;
+        const contest_id = payload.contest_id || 1;
```

Comment:
contest_id is not optional here, no need to default to 1 ... not present is an error